data/simplescreenrecorder.desktop

```diff
@@ -14,4 +15,5 @@ Exec=simplescreenrecorder --logfile
 Terminal=false
 Categories=AudioVideo;Video;Recorder;Qt;
 Keywords=screen recorder;screencast;live streaming;ssr;
+Keywords[es]=grabador de pantalla;transmitir Pantalla;ssr;transmisión en vivo;
```

Comment:
_I consider it useful to add them, although I leave it up to the discretion if it is a necessary change or could be eliminated since a part of the keywords in English are also used within Spanish because they are not original words to the language_